react/features/base/conference/actions.js

```diff
@@ -365,19 +366,45 @@ export function p2pStatusChanged(p2p: boolean) {
 }
 
 /**
+ * Sets a flag for the current JitsiConference to never allow disabling
+ * of audio only mode.
+ *
+ * @returns {Function}
+ */
+export function enableAudioOnlyLock() {
```

Comment:
as we talked about offline, maybe we want this named/done a bit differently.  maybe have `RTCBrowserType.supportsVideo()` and key off of that?